src/server/src/main/java/io/cassandrareaper/metrics/PrometheusMetricsConfiguration.java

```diff
@@ -44,6 +44,8 @@ public static CustomMappingSampleBuilder getCustomSampleMethodBuilder() {
     final Map<String,String> millisSinceLastRepairMetricLabels = new HashMap<>();
     millisSinceLastRepairMetricLabels.put("cluster", "${0}");
     millisSinceLastRepairMetricLabels.put("keyspace", "${1}");
+    millisSinceLastRepairMetricLabels.put("repairid", "${2}");
+    // Legacy label replaced with repairid
```

Comment:
In other metrics same value is mapped to repairid:
https://github.com/thelastpickle/cassandra-reaper/blob/7cc571bbc05f65260aea67012a89c56c803fee66/src/server/src/main/java/io/cassandrareaper/service/RepairRunner.java#L144-L159